contracts/examples/digital-cash/src/deposit_info.rs

```diff
@@ -0,0 +1,13 @@
+use elrond_wasm::{api::BigUintApi, types::TokenIdentifier};
+use elrond_wasm::types::Address;
+
+elrond_wasm::derive_imports!();
+
+#[derive(NestedEncode, NestedDecode, TopEncode, TopDecode, TypeAbi)]
+pub struct DepositInfo<BigUint: BigUintApi>
+{
+	pub amount : BigUint,
```

Comment:
To keep this consistent with the rest of the repo contracts, please write
pub amount: BigUint
instead of
pub amount : BigUint
(no need for the extra space)

Same for the other struct fields.

Reply:
use `cargo fmt` to format your sources, or Ctrl+Shift+I in VSCode to auto-format a file. We try to keep our sources auto-formatted at all times

Reply:
one `cargo fmt` called in the contract folder should fix all issues instantly

Reply:
or even better - configure your VSCode to auto-format on save: https://stackoverflow.com/a/39973431

Reply:
Done!